src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -21,7 +21,6 @@ def create_workflows
 
       parsed_workflows_yaml
         .map { |_workflow_name, workflow_instructions| Workflow.new(workflow_instructions: workflow_instructions, scm_webhook: @scm_webhook, token: @token) }
-        .select(&:valid?)
```

Comment:
We want to get all workflows back to gather potential validation errors of non-valid workflows.